### ISSUE TITLE
Explaining multi-input model

### ISSUE THREAD

**Issue by dmirecki** [TeamHG-Memex/eli5#404]
I am working on deep multi-input classification model (first input is an image, second input is a text). 

Explaining text part while keeping image constant is easy, because in `TextExplainer` I am able to put any function. But now, I would like to explain image part with GradCam, keeping text constant. Is it possible? In `eli5.show_prediction` function I have to put keras model, which is multi-input. How to explain only image part in eli5?

Thanks in advance for any advice!

**Comment by teabolt** [comment]
Hey @dmirecki. If you're using keras can you pass a sublist of `model.layers`, passing only the image layers for explanation?
